js/src/ui/SectionList/sectionList.css

```diff
@@ -39,41 +38,33 @@ $widthExpanded: 42%;
     display: flex;
     justify-content: center;
 
-    /* TODO: As per JS comments, the flex-base could be adjusted in the future to allow for */
+    /* TODO: As per JS comments, the flex-base could be adjusted in the future to allow for
     /* case where <> 3 columns are required should the need arrise from a UI pov. */
     .item {
       box-sizing: border-box;
       display: flex;
       flex: 0 1 $widthNormal;
-      max-width: $widthNormal;
       opacity: 0.85;
       padding: 0.25em;
+
       /* https://www.binarymoon.co.uk/2014/02/fixing-css-transitions-in-google-chrome/ */
       transform: translateZ(0);
       transition: $transition;
+      width: 0;
 
       &:hover {
         opacity: 1;
         z-index: 100;
       }
     }
 
-    &:hover {
-      .item {
-        &.stretchOn {
-          flex: 0 1 $widthShrunk;
-          max-width: $widthShrunk;
-
-          &:hover {
-            flex: 0 0 $widthExpanded;
-            max-width: $widthExpanded;
-          }
-        }
-      }
+    .item.stretchOn:hover {
```

Comment:
There is a reason for the shrinking. When the name (Title) is at exactly 33% and you hover over the row the affected item does not shrink. (This is the original implementation, sadly the simplest, sadly broken in edge-cases)